CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ NOTE: [`epaint`](crates/epaint/CHANGELOG.md), [`eframe`](crates/eframe/CHANGELOG
 * Add `Slider::trailing_fill` for trailing color behind the circle like a `ProgressBar` ([#2660](https://github.com/emilk/egui/pull/2660)).
 
 ### Changed 🔧
+* ⚠️ BREAKING: `Plot::link_axis` and `Plot::link_cursor` now take the name of the group ([#2410](https://github.com/emilk/egui/pull/2410)).
```

Comment:
Move to `## Unreleased`

Reply:
Moved in https://github.com/emilk/egui/pull/2410/commits/ba13680a9d3747fcc2a057bb73e180322b2c9eec